### ISSUE TITLE
Merlin **spoilers ahead obviously***

### ISSUE THREAD

**Issue by Voltaic314** [xdpirate/isitcutshort.com#2]
I don't know if this counts because it's somewhat of a grey area... 

Merlin ended and wasn't cancelled but it does end on a cliffhanger somewhat. 

Let me sum up the ending for you real quick and we'll see what you think: 

The show is about early life Merlin and Prince Arthur, Merlin is ordered to be Prince Arthur's servant for years Merlin discovers he has great magic. The show is basically Merlin going behind Arthur's back constantly to do magic and save Camelot and / or Arthur from demise. 

Finally in the last episode or two, Merlin reveals that he has magic to a dying (now King) Arthur. It's a heartfelt last few minutes of the show but unfortunately it only lasts about half an episode. Eventually Arthur dies shortly after in less than an episode or half an episode... Then Merlin leaves him there and Camelot rots away. Cut to a time screen saying like hundreds of years have passed and it's just Merlin walking along a modern day busy road looking out on where Camelot used to be. And it's teased that one day Arthur will return. 

So basically TLDR the show for rushed due to falling ratings I think in the last season. It basically ended on a cliffhanger. You wait literally 5 seasons for Arthur and Merlin to finally know the truth about Merlin's magic, then he finds out and dies like 30 minutes later and the show basically ends there. lol

**Comment by xdpirate** [comment]
Added!
